fluxacademy.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<head>
+	<title>Flux Academy</title>
+</head>
+<body>
+    <img src="">
+    //sticky nav how to
+        <nav>
+            <ul>
+                <img src="" alt="logo">
+                <li>Courses</li>
+                <li>Student Stories</li>
+                <li>About</li>
+                <li>Log In</li>
+            </ul>
+        </nav>
+    <div>
+        <h1>BECOME A FREELANCE WEB DESIGNER NO MATTER WHERE YOU LIVE</h1>
+            <p>You bring the creativity and passion. We teach you the skills you need to succeed as a freelance web designer. It all happens online, no matter where you are.</p>
```

Comment:
indentation issue

Reply:
I do not see the indentation issue. <h1> is a child of <div> and <p> is child of <h1>.

Reply:
h1 is child of div and p  is child of head

Reply:
please ignore I did a search and it says that p is not a child of head.